docs/using.rst

```diff
@@ -296,9 +296,10 @@ to stdout, like ``RAUC_SYSTEM_SERIAL`` or ``RAUC_SYSTEM_VARIANT``.
 Bundle-Based Customization: Hooks
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-Unlike handlers, hooks are part of the update bundle and must be
-specified in the bundle's :ref:`sec_ref_manifest` file and handled by a common
-executable.
+Unlike handlers, hooks are defined in the update bundle and must be
+specified in the bundle's :ref:`sec_ref_manifest` file.
+All hooks are handled by a common executable that should also be part of the
+bundle.
```

Comment:
```suggestion
All hooks are handled by a common executable that must be included in the
bundle.
```

Reply:
I actually assumed that it would be possible to have absolute paths here. But in fact it isn't really because of the indirect check from https://github.com/rauc/rauc/commit/c3b7c7d64e0891b9712e756fe092fd97f59a2fa0.